eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -131,17 +131,21 @@ jobs:
             - Windows.81.Amd64.Open
             - Windows.10.Amd64.ServerRS5.Open
             - Windows.10.Amd64.Server19H1.Open
+            - Windows.10.Amd64.Server20H2.Open
             - ${{ if ne(parameters.jobParameters.runtimeFlavor, 'mono') }}:
               - (Windows.Nano.1809.Amd64.Open)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-08e8e40-20200107182504
               - (Windows.Server.Core.1909.Amd64.Open)windows.10.amd64.server20h1.open@mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64-20200904200251-272704c
+              - (Windows.Server.Core.20H2.Amd64.Open)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-20H2-helix-amd64-20210922171304-edc3733
           - ${{ if ne(parameters.jobParameters.isFullMatrix, true) }}:
             - Windows.81.Amd64.Open
             - Windows.10.Amd64.Server19H1.ES.Open
+            - Windows.10.Amd64.Server20H2.Open
```

Comment:
We need just a single Windows Servers in isFullMatrix!=true.

Reply:
I would keep the .ES. one on PRs to give us coverage for non-windows?

Reply:
Makes sense to me.